README.md

```diff
@@ -147,6 +147,57 @@ resources:
         newKey: user_id
 ```
 
+### Batch Resources with `properties` paramerters
```

Comment:
Can we move this to API_DOCS.md? Seems more appropriate than the main readme

Reply:
The reason I added it here is, we have `## Batch Resources` in README and `Batch Resources with properties paramerters`  is a feature on the top of batch resources, they are somewhat related. While, API_DOCS is more about how to write a config file and the documentation of each config. 

I think it's okay to leave it in README, but if you think it will make the README too wordy or maybe we don't want to advertise this feature, I could move it to API_DCOS. Please let me know.

Reply:
either API_DOCS.md or could be a seperate file even - this this feels like an important but "power user" feature that doesn't need to be on the homepage